.github/RELEASE.md

```diff
@@ -251,15 +255,66 @@ Currently, there are only 2 activities we use rever for, (1) aggregating the aut
         + 86957814cf235879498ed7806029b8ff5f400034 Updated .authors.yml
         + 3ec7491f2f58494a62f1491987d66f499f8113ad Updated .mailmap
         + 432a9e1b41a3dec8f95a7556632f9a93fdf029fd Updated news
-        + a5c0db938893d2c12cab12a1f7eb3e646ed80373 Updated authorship for 22.9.0
-        + 5e95169d0df4bcdc2da9a6ba4a2561d90e49f75d Updated CHANGELOG for 22.9.0
+        + a5c0db938893d2c12cab12a1f7eb3e646ed80373 Updated authorship for YY.M.0
+        + 5e95169d0df4bcdc2da9a6ba4a2561d90e49f75d Updated CHANGELOG for YY.M.0
         + 93fdf029fd4cf235872c12cab12a1f7e8f95a755 Added first contributions
         ```
 
-8. Push this release branch and open the release PR:
+8. Push this release branch:
 
     ```bash
-    (rever) $ git push -u upstream release-22.9.0
+    (rever) $ git push -u upstream release-YY.M.0
+    ```
+
+9. Open the Release PR.
+
+    <details>
+    <summary>GitHub PR Template</summary>
+
+    ```markdown
+    ### Description
+
+    ✂️ snip snip ✂️ the making of a new release.
+
+    Xref #<RELEASE ISSUE>
     ```
 
+    </details>
+
+10. Update Release Issue to include a link to the Release PR.
+
+11. [Create][new release] the release and **SAVE AS A DRAFT** with the following values:
+
+    > **Note:**
+    > Only publish the release after the Release PR is merged, until then always **save as draft**.
+
+    | Field | Value |
+    |---|---|
+    | Choose a tag | `YY.M.0` |
+    | Target | `main` |
+    | Body | copy/paste blurb from `CHANGELOG.md` |
+
+</details>
+
+### 4. Wait for review and approval of Release PR.
+
+### 5. Merge Release PR and Publish Release.
+
+### 6. Create a new branch (`YY.M.x`) corresponding with the release.
+
+### 7. Open PRs to bump main and conda-forge feedstocks to use `YY.M.0`.
+
+### 8. Hand off to Anaconda's packaging team.
+
+<details>
+<summary>Internal process</summary>
+
+1. Open packaging request in #package_requests, include links to the Release PR and feedstock PRs.
+
+2. Message packaging team/PM to let them know that a release has occurred and that you are the release manager.
```

Comment:
I wonder how we could simplify the release manager announcement, so it becomes a step that the packaging team can look up on their own, instead of having to tell them?

Reply:
discussed and resolved offline

Reply:
Talked on Slack about it, we can resolve this.